content/docs/miscellaneous/acronyms_and_abbreviations.md

```diff
@@ -0,0 +1,85 @@
+---
+# Page title as it appears in the navigation menu
+title: "Acronyms and Abbreviations"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 1
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Acronyms and Abbreviations
+
+AIDA: Artificial Intelligence for Data Analytics
+ALOGIT: name of a Fortran package (see [issue](https://github.com/alan-turing-institute/Hut23/issues/295))
+ASG: AI for Science and Government (the new name of the EPSRC funds from SPF)
```

Comment:
ASG has concluded now.

Reply:
Does that mean it should be removed? It might still be referenced somewhere, and would be useful to know what it stands for